test/output/defaults/output-defaults.test.js

```diff
@@ -18,7 +18,7 @@ describe('output flag defaults', () => {
         });
     });
     it('set default output directory on empty flag', done => {
-        const { stdout, stderr } = run(__dirname, ['--entry', './a.js', '--output'], false);
+        const { stdout, stderr } = run(__dirname, ['--entry', './a.js', '--output', './dist/bundle.js'], false);
```

Comment:
It violates the test statement : "set default output directory on empty flag"

Reply:
Then the output will be `main.js` not `bundle.js`.I think changing it to `main.js` would be a feasible approach ??